lib/document.js

```diff
@@ -10,36 +9,10 @@ export default ({ head, css, html, data, dev, staticMarkup, cdn }) => {
     </head>
     <body>
       <div id='__next' dangerouslySetInnerHTML={{ __html: html }} />
-      {staticMarkup ? null : <script dangerouslySetInnerHTML={{ __html: '__NEXT_DATA__ = ' + htmlescape(data) }} />}
-      {staticMarkup ? null : createClientScript({ dev, cdn })}
+      {staticMarkup ? null : <script dangerouslySetInnerHTML={{
+        __html: `__NEXT_DATA__ =${htmlescape(data)}; module={};`
```

Comment:
I think we need to keep it like this. Since, router.js is expecting it to be commonjs2 (NPM style).
I hope this fix works fine.